Source/App/EncApp/EbAppConfig.c

```diff
@@ -6,12 +6,14 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <sys/stat.h>
 
 #include "EbAppString.h"
 #include "EbAppConfig.h"
 #include "EbAppInputy4m.h"
 
 #ifdef _WIN32
+#include <Fileapi.h>
```

Comment:
```suggestion
#include <windows.h>
```
You need to include windows.h instead of fileapi.h directly since that is where _amd64_ etc are declared

Reply:
Ah, thanks.

Reply:
@kylophone was this change added? The Appveyor Build is still not passing